internal/data/LocalSpiceDbContainer.go

```diff
@@ -0,0 +1,198 @@
+package data
+
+import (
+	"ciam-rebac/internal/conf"
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/authzed/authzed-go/v1"
+	"github.com/go-kratos/kratos/v2/log"
+	"io"
+	"path"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/ory/dockertest"
+)
+
+const (
+	// SpicedbImage is the image used for containerized spiceDB in tests
+	SpicedbImage = "authzed/spicedb"
```

Comment:
image and image tag/version - should be from the config? WDYT?

Reply:
Just looking again, it is in the serviceConfig, but it's hardcoded there. :)

Reply:
If somebody wants to point out a place for me to put it, I'll put it there. Otherwise, we can leave it as is for now. :)

Reply:
dont have a strong opinion, in the sprit of keeping things minimal at start, we can keep it here and think about a test config or spiceDB config at a later point of time.